homework.md

```diff
@@ -3,39 +3,39 @@
 ## Question 1
 
 What is 2 + 2?
+4
 
-5
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+1224
```

Comment:
Hi Bekoo,
Nice work... Just I want to say, I think the correct answer will be 12240.

Reply:
Hi Bekoo, great work, and welcome to the group. Agree with Musa and Sharmaine's suggestions, otherwise really good! Also wonder if you missed the comment about the template we need to use for the pull request title? :)